src/utils/classifier_report.py

```diff
@@ -231,6 +258,122 @@ def create_html_page(self, data, csv_path, stats=False):
                                 toggle_button_txt();
                                 };
                            ''')
+                        # updates review export button
+                        a('''
+                          let updateReviewButton = () => {
+                            let reviewItems = localStorage.getItem("reviewItems");
+                            let len = (reviewItems == null) ? 0 : JSON.parse(reviewItems).length;
+                            $("#reviewExportButton").text(`Export ${len} reviewed items`)
+                            if (len == 0) {
+                                    $("#reviewExportButton").prop("disabled",true);
+                                    $("#buttonsBottom").css("visibility", "hidden");
+
+                            } else {
+                                $("#reviewExportButton").prop("disabled",false);
+                                $("#buttonsBottom").css("visibility", "visible");
+
+                            }
+                          }
+                           ''')
+                        # loads review choices from local storage
+                        a('''
+                          let loadReviewChoices = () => {
+                          if (!localStorage.getItem('reviewItems')) {
+                            localStorage.setItem("reviewItems", "[]");  
+                          } else {
+                            let reviewedItems = JSON.parse(localStorage.getItem("reviewItems"))
+                            $('.reviewButton').removeClass('active');
+                            reviewedItems.forEach(e => {
+                                e = JSON.parse(e);
+                               if ($(`#${e['id']}`).hasClass('active') === false) {
+                                  $(`#${e['id']}`).toggleClass('active');
+                                }
+                            })
+                          }
+                        }
+                           ''')
+                        # initialize localStorage object
+                        a('''
+                        if (localStorage.length == 0) {

```

Comment:
Is the goal here to set reviewItems if its not present at page load time? If I run `localStorage.length` in my console it returns 4 since my localStorage has other stuff set from elsewhere on dcr-test. So I think this should either directly check for the reviewItems key to be set, or it isn't necessary since the page loaded without errors for me after i cleared reviewItems from the local storage